internal/local/structure.go

```diff
@@ -340,6 +353,21 @@ func writeServices(rootDir string, services []ServiceStructure) error {
 		); err != nil {
 			return err
 		}
+
+		// Default Rule
+		defaultRule, err := MarshalJSON(svc.DefaultRule)
```

Comment:
what gets written here if `DefaultRule` is nil or empty?  do we want the file to exist but be empty/`{}`?

I would have guessed maybe we want a `if len(svc.DefaultRule) != 0` check wrapping this file write

Reply:
Good catch! Added this check as well as a test case to ensure this is handled properly